src/main/java/ch/ethz/matsim/baseline_scenario/additional_traffic/freight/run/RunFreightTrafficCreator.java

```diff
@@ -0,0 +1,57 @@
+package ch.ethz.matsim.baseline_scenario.additional_traffic.freight.run;
+
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.FreightTraffic;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.FreightTrafficCreator;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.items.FreightTrafficODItem;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.readers.CumulativeDepartureProbabilityReader;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.readers.FreightTrafficODReader;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.readers.ZoneReader;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.utils.DepartureTimeGenerator;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.utils.FreightFacilitySelector;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.writers.FreightFacilitiesWriter;
+import ch.ethz.matsim.baseline_scenario.additional_traffic.freight.writers.FreightPopulationWriter;
+import org.matsim.facilities.ActivityFacilities;
+import org.matsim.facilities.ActivityFacility;
+
+import java.util.*;
+
+public class RunFreightTrafficCreator {
+    public static void main(String[] args) {
+        final String zoneCoordinatesFile = args[0];
+        final String scenarioFacilitiesFile = args[1]; // all scenario facilities incl secondary facilities and bc facilities.
+        final String utilityVehiclesFile = args[2];
+        final String trucksFile = args[3];
+        final String tractorTrailersFile = args[4];
+        final String cumulativeProbabilityFreightDeparturesFile = args[5];
+        final double scalingFactor = Double.parseDouble(args[6]); // for example for a 1% population enter "0.01"
+        final int randomSeed = Integer.parseInt(args[7]);
+        final String outputFacilities = args[8];
+        final String outputPopulation = args[9];
+
+        Random random = new Random(randomSeed);
+
+        DepartureTimeGenerator departureTimeGenerator = new DepartureTimeGenerator(random,
+                (new CumulativeDepartureProbabilityReader().read(cumulativeProbabilityFreightDeparturesFile)));
+
+
+        // TODO : HOW DO I SIMPLY LOAD ACTIVITY FACILITIES?
```

Comment:
Scenario scenario = ScenarioUtils.createScenario();
new ActivityReader(scenario).read(PATH);

something like that